cves/CVE-2011-3944.yml

```diff
@@ -342,7 +342,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: The major mistake in this CVE was the lack of having files being scanned for malicious content. I do not belive that this is a coding mistake, but rather a mistake of not thinking of all of the possible ways that input could be read into the system.
```

Comment:
Since this was introduced in 2006, do you think the developers might just not have known better? We've come a long way in security since then.

Reply:
I think it was just a flaw they didn't think cause all they added was an if statement